infra/src/test/kotlin/fi/fta/geoviite/infra/publication/ValidationContextIT.kt

```diff
@@ -0,0 +1,191 @@
+package fi.fta.geoviite.infra.publication
+
+import fi.fta.geoviite.infra.DBTestBase
+import fi.fta.geoviite.infra.common.IntId
+import fi.fta.geoviite.infra.common.KmNumber
+import fi.fta.geoviite.infra.geocoding.GeocodingService
+import fi.fta.geoviite.infra.switchLibrary.SwitchLibraryService
+import fi.fta.geoviite.infra.tracklayout.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+
+@ActiveProfiles("dev", "test")
+@SpringBootTest
+class ValidationContextIT @Autowired constructor(
```

Comment:
Vähän uusia testejä kontekstille. Kirjoittelen näitä vielä vähän lisää, mutta ajattelin että voi jo nyt pistää reviewiin tämän tarkaisun